playbook/createGame.ts

```diff
@@ -0,0 +1,80 @@
+import { task } from 'hardhat/config';
+import { RankifyDiamondInstance, Rankify } from '../types';
+import { IRankifyInstanceCommons } from '../types/src/facets/RankifyInstanceMainFacet';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+task('createGame', 'Create new game')
+  .addOptionalParam('gameCreator', 'Player address who will create game')
+  .setAction(
+    async ({
+      gameCreator,
+    }: { gameCreator: string;},
+     hre: HardhatRuntimeEnvironment) => {
+      const { deployments, getNamedAccounts } = hre;
+      const rankifyDeployment = await deployments.get('Rankify');
+      const rankifyInstanceDeployment = await deployments.get('RankifyInstance');
+      const { owner, registrar, defaultPlayer } = await getNamedAccounts();
+      gameCreator = gameCreator ?? defaultPlayer;
+
+      await mintTokensToGameCreator(rankifyDeployment, gameCreator, owner, hre);
+      await approveTokensUse(rankifyDeployment, rankifyInstanceDeployment, gameCreator, hre);
+      await createGame(rankifyInstanceDeployment, gameCreator, registrar, hre);
+    },
+  );
+
+export const mintTokensToGameCreator = async (
+  rankifyDeployment: any,
+  gameCreator: string,
+  owner: string,
+  hre: HardhatRuntimeEnvironment
+) => {
+  const rankifyContract = new hre.ethers.Contract(
+    rankifyDeployment.address,
+    rankifyDeployment.abi,
+    hre.ethers.provider.getSigner(owner),
+  ) as Rankify;
+  const tx = await rankifyContract
+    .mint(gameCreator, hre.ethers.utils.parseEther('1000'));
+  console.log(await tx.wait().then((r: any) => r.logs));
+};
+
+export const approveTokensUse = async (
+  rankifyDeployment: any,
+  rankifyInstanceDeployment: any,
+  gameCreator: string,
+  hre: HardhatRuntimeEnvironment
+) => {
+  const rankifyContract = new hre.ethers.Contract(
+    rankifyDeployment.address,
+    rankifyDeployment.abi,
+    hre.ethers.provider.getSigner(gameCreator),
+  ) as Rankify;
+
+  const tx = await rankifyContract
+    .approve(rankifyInstanceDeployment.address, hre.ethers.constants.MaxUint256);
+  console.log(await tx.wait().then((r: any) => r.logs));
+};
+
+export const createGame = async (
```

Comment:
Given the playbook narrative,  i think it would be more clear if we describe state rather than action. 

I.e "gameCreated" and "DomainInitialized" 

With this, it is easier to understand chain actions. 
For example you would need only one call for "GameCreated" in order to reproduce state, everything related to domains initialization, username registration etc would be already enforced within that scenario.